huaweicloud/services/cse/resource_huaweicloud_cse_microservice_engine.go

```diff
@@ -26,7 +26,14 @@ import (
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
 )
 
-var DefaultVersion = "CSE2"
+var (
+	DefaultVersion = "CSE2"
+
+	EngineNotFoundCodes = []string{
```

Comment:
It is best to change the first letter to lowercase.

Reply:
fixed.